src/main/webapp/app/genomic/genomic.component.ts

```diff
@@ -31,7 +31,7 @@ export class GenomicComponent implements OnInit {
     'protein_altering', 'splice site_mutation', 'stop_retained', 'synonymous', '3\'UTR', '3_prime_UTR',
     '5\'Flank', '5\'UTR', '5\'UTR_mutation', '5_prime_UTR'];
     annotated_variants = this.trialService.getOncokbVariants();
-    oncokb = environment.oncokb ? environment.oncokb : false;
```

Comment:
Same here, we need to support setting from environment

Reply:
This oncokb will fetch value from trial.service.ts, so we just need to maintain only one config variable.